apps/desktop/src/lib/shared/LazyloadContainer.svelte

```diff
@@ -37,10 +37,14 @@
 		const containerChildren = lazyContainerEl.children;
 		if (containerChildren.length < minTriggerCount) return;
 
-		intersectionObserver.observe(containerChildren[containerChildren.length - 1] as HTMLElement);
+		const lastChild = containerChildren[containerChildren.length - 1];
+		if (!lastChild) return;
+
+		intersectionObserver.observe(lastChild);
 	}
 
 	onMount(() => {
+		attachIntersectionObserver();
```

Comment:
Why is it required that we call `attachIntersectionObserver` here when we have the $effect() block on line 18 through 23 which should be starting the observation?

Reply:
It seems that it doesn't always get called, weirdly. 

The history board was mounted and everything was setup, but it appears that the container didn't mutate and hence the intersection observer didn't start its observation

Reply:
Maybe there's a better way of fixing this, though

Reply:
I see. `attachIntersectionObserver` expects `lazyContainerEl` to be present (the type signature of `lazyContainerEl` ought to be `HTMLDivElement | undefined`).

We could quite reasonably call `attachIntersectionObserver` after line 20.